app/src/main/java/com/example/eggert_hoppens_project2/UserInfo.java

```diff
@@ -20,10 +27,10 @@ public UserInfo(String userName, String userPassword, boolean isAdmin) {
     @Override
     public String toString() {
         return "UserInfo{" +
-                "mUserName='" + mUserName + '\'' +
-                ", mUserPassword='" + mUserPassword + '\'' +
-                ", mUserId=" + mUserId +
-                ", mIsAdmin=" + mIsAdmin +
+                "UserName='" + mUserName + '\'' +
+                ", UserPassword='" + mUserPassword + '\'' +
+                ", UserId=" + mUserId +
```

Comment:
Probably should remove this from the toString() method. I also forgot to remove it. Unless you want to see it for now as a for ease of debugging, then change it later.

Reply:
Yeah, once we debug and get everything in line with that, I'll remove it.